DocsCode/NavigationTutorial/NavigationTutorialViewController.m

```diff
@@ -0,0 +1,150 @@
+// #-code-snippet: navigation dependencies-objc
+#import "NavigationTutorialViewController.h"
+
+@import Mapbox;
+@import MapboxCoreNavigation;
+@import MapboxNavigation;
+@import MapboxDirections;
+// #-end-code-snippet: navigation dependencies-objc
+
+@interface NavigationTutorialViewController () <MGLMapViewDelegate>
+
+@property (nonatomic) MBNavigationMapView *mapView;
+// #-code-snippet: navigation directions-route-objc
+@property (nonatomic) MBRoute *directionsRoute;
+// #-end-code-snippet: navigation directions-route-objc
+
+
+@end
+
+@implementation NavigationTutorialViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // #-code-snippet: navigation init-map-objc
+    self.mapView = [[MBNavigationMapView alloc] initWithFrame:self.view.bounds];
+
+    [self.mapView setCenterCoordinate:CLLocationCoordinate2DMake(30.265, -97.741)
+                            zoomLevel:11 animated:NO];
+    [self.view addSubview:self.mapView];
+    // Set the map view's delegate
+    self.mapView.delegate = self;
+    // #-end-code-snippet: navigation init-map-objc
+    
+    // #-code-snippet: navigation user-location-objc
+    // Allow the map view to display the user's location
+    self.mapView.showsUserLocation = YES;
+    // #-end-code-snippet: navigation user-location-objc
+    
+    // #-code-snippet: navigation gesture-recognizer-objc
+    // Add a gesture recognizer to the map view
+    UITapGestureRecognizer *setDestination = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:@selector(didLongPress:)];
```

Comment:
This incorrectly initializes to a `UITapGestureRecognizer ` to `UILongPressGestureRecognizer`.